plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergParquetConnectorTest.java

```diff
@@ -22,4 +24,22 @@ public TestIcebergParquetConnectorTest()
     {
         super(PARQUET);
     }
+
+    @Override
+    protected boolean supportsRowGroupStatistics(String typeName)
+    {
+        return !(typeName.equalsIgnoreCase("varbinary") ||
+                typeName.contains("decimal") ||
+                typeName.contains("time"));
+    }
+
+    @Override
+    protected Session withSmallRowGroups()
+    {
+        return Session.builder(getSession())
+                .setCatalogSessionProperty("iceberg", "parquet_writer_page_size", "100B")
+                .setCatalogSessionProperty("iceberg", "parquet_writer_block_size", "100B")
+                .setCatalogSessionProperty("iceberg", "parquet_writer_row_group_max_row_count", "10")
```

Comment:
would it be enough to set parquet_writer_row_group_max_row_count only?
ie are all 3 needed?

Reply:
We need both, the naming of the `DEFAULT_ROW_GROUP_MAX_ROW_COUNT` was a bit misleading. It's being used as the number of rows to pass per Page chunk into the Parquet writer, but doesn't control the row group size the writer decides to use. 

It's actually the minimum number of rows in a row group, because the writer won't split the Page f it goes over the Page size.

Reply:
Not that I fully understand it. But it looks like we should change the name if it is misleading.

Reply:
`parquet_writer_batch_size`?

Reply:
Yeah - better IMO

Reply:
Renamed, thanks